generators/gae/index.js

```diff
@@ -555,8 +556,9 @@ module.exports = class extends BaseGenerator {
                 this.log(chalk.bold('\nCreating New Cloud SQL Instance'));
 
                 const name = this.gcpCloudSqlInstanceName;
+                const dbVersion = this.prodDatabaseType === 'postgresql' ? ' --database-version="POSTGRES_9_6" --tier="db-g1-small"' : '';
```

Comment:
for mysql, it uses the default tier. can we keep that the same for postgres?

also, would it be more clear to change `dbVersion` to `dbVersionFlag`.

Reply:
We can't keep default tier for postgres, as described in doc. I tried but faced an explicit error then.

Yes dbVersionFlag may be better

Reply:
done